### ISSUE TITLE
Getting this error multiple times a day. Kodi is running 24h. 

### ISSUE THREAD

**Issue by firstdominator** [boralyl/kodi-recently-added#11]
.
This error originated from a custom integration.

Logger: custom_components.kodi_recently_added.entities
Source: custom_components/kodi_recently_added/entities.py:44
Integration: Kodi Recently Added Media (documentation, issues)
First occurred: November 26, 2021, 6:14:17 PM (15 occurrences)
Last logged: 1:00:48 AM

Error updating sensor, is kodi running?
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/jsonrpc_websocket/jsonrpc.py", line 141, in wait
    await self._event.wait()
  File "/usr/local/lib/python3.9/asyncio/locks.py", line 226, in wait
    await fut
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/jsonrpc_websocket/jsonrpc.py", line 49, in send_message
    response = await pending_message.wait(self._timeout)
  File "/usr/local/lib/python3.9/site-packages/jsonrpc_websocket/jsonrpc.py", line 142, in wait
    return self._response
  File "/usr/local/lib/python3.9/site-packages/async_timeout/__init__.py", line 45, in __exit__
    self._do_exit(exc_type)
  File "/usr/local/lib/python3.9/site-packages/async_timeout/__init__.py", line 92, in _do_exit
    raise asyncio.TimeoutError
asyncio.exceptions.TimeoutError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/config/custom_components/kodi_recently_added/entities.py", line 44, in async_update
    result = await self.kodi.call_method(
  File "/usr/local/lib/python3.9/site-packages/pykodi/kodi.py", line 294, in call_method
    return await getattr(self._server, method)(**kwargs)
  File "/usr/local/lib/python3.9/site-packages/jsonrpc_websocket/jsonrpc.py", line 55, in send_message
    raise TransportError('Transport Error', message, exc)
jsonrpc_base.jsonrpc.TransportError: ("Error calling method 'VideoLibrary.GetRecentlyAddedMovies': Transport Error", TimeoutError())

**Comment by boralyl** [comment]
I took a look at how the core kodi component handles Transport Errors and it logs them as warnings instead of errors.  So I updated the custom component in [the latest release](https://github.com/boralyl/kodi-recently-added/releases/tag/v2.0.3) to do the same.  So you should no longer see these as errors in your logs.

**Comment by firstdominator** [comment]
Thanks a lot. 
Quick question, is there any reason why this is a warning ? What does it means really? Do you have any more details.

**Comment by boralyl** [comment]
It just means there was an error communicating with the kodi server, which could happen for a number of reasons.  In my setup I run kodi on an nvidia shield.  After some time when kodi is in the bakground, android will clean it up and it will no longer be running which causes the warnings in my HA instance.  When I launch kodi again the component works again as expected.

The core kodi component logs a warning, but it's only when the `call_method` function is called.  This would normally be something you would do manually with a service call, whereas this component calls it at an interval.  Since most of the time this is an intermittent issue, I may just change the log level to debug to hide the message unless debug logging is enabled for the component.  The reason being is that it's really not something a user can resolve (unless their kodi server is actually down).



TLDR: I think I may just make this a debug log instead of a warning.
